Gemfile.lock

```diff
@@ -443,5 +443,8 @@ DEPENDENCIES
   webmock
   xray-rails
 
+RUBY VERSION
```

Comment:
No, not at all.  This was not intended.  Sorry, this is my first time doing a pull request.  I was trying to manually remove the changed blocks, but seems like I didn't clear it properly.